Cheza-Dzabala/way-farer
# chore-create confirmation boxes

#### What does this PR do?
Incorporates dialog boxes for confirmation of actions in the UI

#### Description of Task to be completed?

Provide feedback in the UI in the form of a notification when a trip is cancelled by the admin. If the cancellation is successful, a popup should appear that notifies the admin of the successful cancellation. If it fails, a notification should appear that notifies the admin of the failure to cancel. Furthermore, a dialog should appear that asks the admin to confirm the cancellation before the trip is cancelled.

#### How should this be manually tested?

Log in via Github pages with the account:

```javascript

{
  "email": "cheza@gmail.com",
  "password": "Runfree8418_!*"
}

```
Go into the admin panel. Select view scheduled trips and attempt to cancel a trip. It should present a dialog box with the options "Yes, I'm sure" and "On second thought, no"

#### Any background context you want to provide?

Built with css to emulate modals in the UI. 

#### What are the relevant pivotal tracker stories?

#### #167724327

#### Screenshots (if appropriate)
<img width="1704" alt="Screenshot 2019-08-06 at 11 33 26 AM" src="https://user-images.githubusercontent.com/13886438/62529050-2df49e80-b83e-11e9-94b8-f26ea65f9a7c.png">
<img width="1652" alt="Screenshot 2019-08-06 at 11 33 56 AM" src="https://user-images.githubusercontent.com/13886438/62529051-2e8d3500-b83e-11e9-9705-8f794ec970df.png">
<img width="1909" alt="Screenshot 2019-08-06 at 11 34 07 AM" src="https://user-images.githubusercontent.com/13886438/62529054-2e8d3500-b83e-11e9-88a8-ad1d4c19c039.png">



#### Questions:

none
